addons/udes_stock/tests/test_limit_orderpoints.py

```diff
@@ -0,0 +1,56 @@
+"""Tests for the limit orderpoints flag."""
+
+from odoo.exceptions import ValidationError
+from . import common
+
+
+class LimitOrderpointsTestCase(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super(LimitOrderpointsTestCase, cls).setUpClass()
+        # Create an orderpoint with values
+        cls.op = cls.create_orderpoint(cls.apple, cls.test_output_location_01,
+                                       5, 10)
+
+    def test01_cannot_add_orderpoint_if_one_exists(self):
+        """ Tests cannot add an orderpoint if one already exists and the limit
+        orderpoints flag is set on the location.
+        """
+        self.test_output_location_01.u_limit_orderpoints = True
+        with self.assertRaises(ValidationError) as cm:
+            self.create_orderpoint(self.banana,
+                                   self.test_output_location_01,
+                                   5, 10)
+        ex = cm.exception
+        self.assertEqual('An order point for location Test output location 01 '
```

Comment:
Does it worth to replace `Test output location 01` by  `%s` and use `self.test_output_location_01`? in case the name of the location changes the test is not going to fail since we don't care about the name of the location.